From 5f173fa7d87c998dd912998c839c53556f69c6c8 Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Fri, 11 Aug 2017 11:59:16 +0800
Subject: [PATCH] ARM: dts: rk3288-evb: set "system-power-controller" for
 act8846

PMIC should set "system-power-controller" in dts, otherwise
it couldn't be closed when system shutdown.

Change-Id: I3efd7efb1b1911621f21e8a741e2f6c2eebb60dc
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-act8846.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-act8846.dts b/arch/arm/boot/dts/rk3288-evb-act8846.dts
index 7286243486e2..be3c03283b8d 100644
--- a/arch/arm/boot/dts/rk3288-evb-act8846.dts
+++ b/arch/arm/boot/dts/rk3288-evb-act8846.dts
@@ -264,6 +264,7 @@
 	act8846: act8846@5a {
 		compatible = "active-semi,act8846";
 		reg = <0x5a>;
+		system-power-controller;
 		status = "okay";
 
 		vp1-supply = <&vcc_sys>;
-- 
2.35.3

